src/keylessBackup/SignInWithEmail.tsx

```diff
@@ -1,27 +1,99 @@
 import { NativeStackScreenProps } from '@react-navigation/native-stack'
-import React, { useState } from 'react'
+import React, { useRef, useState } from 'react'
 import { useTranslation } from 'react-i18next'
-import { SafeAreaView, ScrollView, StyleSheet, Text } from 'react-native'
+import { Image, ScrollView, StyleSheet, Text, View } from 'react-native'
 import { useAuth0 } from 'react-native-auth0'
+import { SafeAreaView, useSafeAreaInsets } from 'react-native-safe-area-context'
 import { KeylessBackupEvents } from 'src/analytics/Events'
 import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import BackButton from 'src/components/BackButton'
+import BottomSheet, { BottomSheetRefType } from 'src/components/BottomSheet'
 import Button, { BtnSizes, BtnTypes } from 'src/components/Button'
+import CustomHeader from 'src/components/header/CustomHeader'
 import GoogleIcon from 'src/icons/Google'
+import { email } from 'src/images/Images'
 import KeylessBackupCancelButton from 'src/keylessBackup/KeylessBackupCancelButton'
 import { googleSignInCompleted, keylessBackupStarted } from 'src/keylessBackup/slice'
-import { KeylessBackupFlow } from 'src/keylessBackup/types'
-import { emptyHeader } from 'src/navigator/Headers'
+import { KeylessBackupFlow, KeylessBackupOrigin } from 'src/keylessBackup/types'
+import { HeaderTitleWithSubtitle } from 'src/navigator/Headers'
 import { navigate } from 'src/navigator/NavigationService'
 import { Screens } from 'src/navigator/Screens'
 import { StackParamList } from 'src/navigator/types'
-import { useDispatch } from 'src/redux/hooks'
+import {
+  getOnboardingStepValues,
+  goToNextOnboardingScreen,
+  onboardingPropsSelector,
+} from 'src/onboarding/steps'
+import { useDispatch, useSelector } from 'src/redux/hooks'
 import Colors from 'src/styles/colors'
 import { typeScale } from 'src/styles/fonts'
 import { Spacing } from 'src/styles/styles'
+import variables from 'src/styles/variables'
 import Logger from 'src/utils/Logger'
 
 const TAG = 'keylessBackup/SignInWithEmail'
 
+function SignInWithEmailBottomSheet({
+  keylessBackupFlow,
+  origin,
+  bottomSheetRef,
+}: {
+  keylessBackupFlow: KeylessBackupFlow
+  origin: KeylessBackupOrigin
+  bottomSheetRef: React.RefObject<BottomSheetRefType>
+}) {
+  const { t } = useTranslation()
+  const onboardingProps = useSelector(onboardingPropsSelector)
+  const onPressContinue = () => {
+    ValoraAnalytics.track(KeylessBackupEvents.cab_setup_recovery_phrase)
+    bottomSheetRef.current?.close()
+    goToNextOnboardingScreen({
+      firstScreenInCurrentStep: Screens.SignInWithEmail,
+      onboardingProps: { ...onboardingProps, showRecoveryPhraseEducation: true },
+    })
```

Comment:
per comment in steps, this can just navigate directly

Reply:
Fixed in 03854844f6b6f9834fda241e3761180f23644925!